launcher/minecraft/mod/ModFolderModel.cpp

```diff
@@ -217,7 +213,7 @@ bool ModFolderModel::stopWatching()
     return ResourceFolderModel::stopWatching({ m_dir.absolutePath(), indexDir().absolutePath() });
 }
 
-auto ModFolderModel::selectedMods(QModelIndexList& indexes) -> QList<Mod*>
+auto ModFolderModel::selectedMods(const QModelIndexList& indexes) -> QList<Mod*>
```

Comment:
Not relevant to your changes, but what's up with the prolific use of trailing return types? I don't even see a situation that needs them.